services/web-app/pages/contributing/schema.mdx

```diff
@@ -1,17 +1,34 @@
-# Resource schemas
+---
+title: Schema
+description:
+  Carbon resources (e.g. libraries, assets, design kits) follow a schema to standardize definitions
+  and documentation. This structured information is indexed by Carbon's platform to use in the
+  website and other services.
+---
 
-Carbon Design System resources (e.g. libraries, assets, design kits) follow a schema to document
-each resource in a standardized way. This structured information is indexed by Carbon's platform to
-use in the website and other services.
+<PageDescription>
 
-Schemas are applied through metadata files written in YAML. If you're new to YAML and want to learn
-more, see "[Learn YAML in Y minutes](https://learnxinyminutes.com/docs/yaml)."
+Carbon resources (e.g. libraries, assets, design kits) follow a schema to standardize definitions
+and documentation. This structured information is indexed by Carbon's platform to use in the website
+and other services.
 
-**Example**
+</PageDescription>
 
-carbon.yml
+<AnchorLinks>
 
-```yml
+<AnchorLink>Stable version</AnchorLink>
+<AnchorLink>Schema keys</AnchorLink>
+<AnchorLink>Library schema</AnchorLink>
+<AnchorLink>Asset schema</AnchorLink>
+<AnchorLink>Design kit schema</AnchorLink>
+<AnchorLink>Shared schemas</AnchorLink>
+
+</AnchorLinks>
+
+Schemas are applied through metadata files written in YAML. If you're new to YAML and want to learn
```

Comment:
I think we could expand the text here a little to have a softer introduction. Perhaps adding something like:

Maintainers can control content and resource visibility through a metadata file that follows a clear schema. This helps us document each resource in a standardized way and enables everything to be indexed by Carbon’s new underlying platform. Below is the latest stable version of the schema.